js/packages/mobile-wallet-adapter-protocol/src/transact.ts

```
@@ -326,10 +334,31 @@ export async function transactRemote<TReturn>(
     callback: (wallet: RemoteMobileWallet) => TReturn,
     config: RemoteWalletAssociationConfig,
 ): Promise<{associationUrl: URL, result: Promise<TReturn>}> {
+    return startRemoteScenario(config).then((scenario) => {
+        return { 
+            associationUrl: scenario.associationUrl, 
+            result: scenario.wallet.then((wallet) =>{
```

Noticing a `.then` but no `.catch`.

- If there is an error starting the remote scenario, does that bubble up somewhere else?
- Is the behavior any different than when an error occurs starting a local scenario?

---

its just calling the underlying method that holds the actual logic, so its mimicking that behavior. A catch clause here would need to rethrow the error so it can be caught by the caller (see wallet.ts) so it owuld be pointless. Unless I am missing somehting. 

Also `transactRemote` has been deleted in my refactor branch so moot point regardless.